docs/ops/pooling/MaxPool_8.md

```diff
@@ -0,0 +1,372 @@
+  * `auto_pad = explicit` and `rounding_type = floor`
+        `H_out = floor((H + pads_begin[0] + pads_end[0] - ((kernel[0] - 1) * dilations[0] + 1)) / strides[0] + 1)`
+        `W_out = floor((W + pads_begin[1] + pads_end[1] - ((kernel[1] - 1) * dilations[1] + 1)) / strides[1] + 1)`
+        `D_out = floor((D + pads_begin[2] + pads_end[2] - ((kernel[2] - 1) * dilations[2] + 1)) / strides[2] + 1)`
+
+  * `auto_pad = explicit` and `rounding_type = ceil`
+        `H_out = ceil((H + pads_begin[0] + pads_end[0] - ((kernel[0] - 1) * dilations[0] + 1)) / strides[0] + 1)`
+        `W_out = ceil((W + pads_begin[1] + pads_end[1] - ((kernel[1] - 1) * dilations[1] + 1)) / strides[1] + 1)`
+        `D_out = ceil((D + pads_begin[2] + pads_end[2] - ((kernel[2] - 1) * dilations[2] + 1)) / strides[2] + 1)`
+
+  * `auto_pad = valid`
+        `H_out = ceil((H - ((kernel[0] - 1) * dilations[0] + 1) + 1) / strides[0])`
+        `W_out = ceil((W - ((kernel[1] - 1) * dilations[1] + 1) + 1) / strides[1])`
+        `D_out = ceil((D - ((kernel[2] - 1) * dilations[2] + 1) + 1) / strides[2])`
+
+  * `auto_pad = same_upper/same_lower`
+        `H_out = H`
+        `W_out = W`
+        `D_out = D`
+
+
+If `H + pads_begin[i] + pads_end[i] - kernel[i]` is not divisible by `strides[i]` evenly then the result is rounded with the respect to `rounding_type` attribute.
+
+Example 1 shows how *MaxPool* operates with 4D input using 2D kernel and `auto_pad = explicit`
+
+```
+input = [[[[-1, 2, 3],
+           [4, 5, -6],
+           [-7, 8, 9]]]]
+strides = [1, 1]
+pads_begin = [1, 1]
+pads_end = [1, 1]
+kernel = [2, 2]
+rounding_type = "floor"
+auto_pad = "explicit"
+output0 = [[[[-1, 2, 3, 3],
+             [4, 5, 5, -6],
+             [4, 8, 9, 9],
+             [-7, 8, 9, 9]]]]
+output1 = [[[[0, 1, 2, 2],
+             [3, 4, 4, 5],
+             [3, 7, 8, 8],
+             [6, 7, 8, 8]]]]
+```
+
+Example 2 shows how *MaxPool* operates with 3D input using 1D kernel and `auto_pad = valid`
+
+```
+input = [[[-1, 2, 3, 5, -7, 9, 1]]]
+strides = [1]
+kernel = [3]
+rounding_type = "floor"
+auto_pad = "valid"
+output0 = [[[3, 5, 5, 9, 9]]]
+output1 = [[[2, 3, 3, 5, 5]]]
+```
+
+Example 3 shows how *MaxPool* operates with 4D input using 2D kernel and `auto_pad = same_lower`
+
+```
+input = [[[[-1, 2, 3],
+         [4, 5, -6],
+         [-7, 8, 9]]]]
+strides = [1, 1]
+kernel = [2, 2]
+rounding_type = "floor"
+auto_pad = "same_lower"
+output0 = [[[[-1, 2, 3],
+            [4, 5, 5]
+            [4, 8, 9]]]]
+output1 = [[[[0, 1, 2],
+            [3, 4, 4]
+            [3, 7, 8]]]]
+```
+
+Example 4 shows how *MaxPool* operates with 4D input using 2D kernel and `auto_pad = same_upper`
+
+```
+input = [[[[-1, 2, 3],
+           [4, 5, -6],
+           [-7, 8, 9]],
+          [[2, -1, 5],
+           [6, -7, 1],
+           [8, 2, -3]]]]
+strides = [1, 1]
+kernel = [2, 2]
+rounding_type = "floor"
+auto_pad = "same_upper"
+output0 = [[[[5, 5, 3],
```

Comment:
You have fixed a bug in the specification. Please, update the MaxPool_1.md as well.